src/components/annotator/AppellationCreator.vue

```diff
@@ -105,7 +169,20 @@ export default class AppellationCreator extends Vue {
 				}
 			},
 		);
+		this.$store.watch(
+			(state, getters) => getters.getAnnotatorisDateAppellation,
+			(newValue, oldValue) => {
+				if (newValue === true) {
```

Comment:
can you just do if(newValue)? That statement should only run if its true and if newValue is true then there shouldn't be a probably.